### DISCUSSION TITLE
Include path, intelliSenseMode and compiler help

### DISCUSSION THREAD

**Discussion by bigpolarbear777** [microsoft/vscode-cpptools#8618] [Q&A]
I am new to arduino and VS code so I don't know to much of how it works. I've been trying to fix this problem for a long time, tried every solution the internet had to offer. I have the arduino ide, extension and the c++ extension.

I want to be able to have all the libraries in my computer included in VS but don't know how to do it. I'm very new and don't know what type of files I can include. I also want to have the correct intellisensemode and compiler.
I have arduino 1.8.19 and VS code 1.63.2
![Screenshot 2022-01-01 012857](https://user-images.githubusercontent.com/96931753/147845229-53980e2a-d85e-4821-ac20-bdef97a2e024.jpg)
plz help

**Comment by sean-mcmanus** [comment]
What problem are you having exactly? Is it a problem compiling or with IntelliSense? What is the error? What is the output of running C/C++: Log Diagnostics?

**Reply by bigpolarbear777** [reply]
Hello, I am having the problem with both. For the output I think its this, whenever I try to verify if my code works it Output path is not specified Unable to reuse previously compiled files. Build will be slower. See README. Loading configuration... 
Also the commands in my code are underlined red, stating its undefine
![Screenshot 2022-01-05 183450](https://user-images.githubusercontent.com/96931753/148304858-8ce20e9f-77e5-4fda-a935-b2d29815c64b.jpg)
d

**Reply by bigpolarbear777** [reply]
I forgot to mention that I can't figure out what or where is the correct path name is to put in the include file. I know where the library I want I include is, but whenever I type its location on the include section in the setting tab, my include command still wont work. I don't know if its because I used the wrong path name or something else.

**Reply by sean-mcmanus** [reply]
Having you tried asking at the Arduino extension? I'm not familiar with how to use the Arduino extension.
